filebeat/module/postgresql/log/manifest.yml

```
@@ -4,10 +4,16 @@ var:
   - name: paths
     default:
       - /var/log/postgresql/postgresql-*-*.log*
+      - /var/log/postgresql/postgresql-*-*.csv
```

Add a wildcard if these files can be rotated.
```suggestion
      - /var/log/postgresql/postgresql-*-*.csv*
```